### PR TITLE
Demo react native

### PR DESCRIPTION
**Pull request by JeandeCampredon**

### CODE REVIEW

**Review by FabienGreard — CHANGES_REQUESTED** [review]
Is v2 (now) blocking somewhere ? Why would you use v1 ?

File: packages/app/src/components/Home.js [hunk 1]
```diff
@@ -136,6 +136,21 @@ const textSection = css`
   }
 `;
 
+const nextDates = [
```
Comment (FabienGreard) [inline packages/app/src/components/Home.js]: Could be great to link meetup date OR maybe just do each 2 weeks than a simple array of dates

File: packages/app/src/serviceWorker.js [hunk 2]
```diff
@@ -1,26 +1,67 @@
...
+}
+
+function update(request) {
+  console.log('update start');
+  return caches.open(CACHE).then(function(cache) {
+    return Promise.all([fetch(request), fromCache(request)]).then(function([
+      newResponse,
+      oldResponse,
+    ]) {
+      const n = newResponse ? newResponse.clone() : { text: () => Promise.resolve(null) };
```
Comment (FabienGreard) [inline packages/app/src/serviceWorker.js:43]: const => var

Comment (JeandeCampredon) [reply packages/app/src/serviceWorker.js:43]: 👍

File: packages/app/src/components/Home.js (hunk 1 above)
Comment (JeandeCampredon) [reply packages/app/src/components/Home.js]: 👍

**Review by FabienGreard — APPROVED** [review]

**Pull request merged**